controllers/v1/platform-roles.js

```diff
@@ -1,23 +1,22 @@
 /**
- * name : platformRolesExt.js
+ * name : platform-roles.js
```

Comment:
@rakeshSgr - Let's have the consistency of camel case everywhere

Reply:
resolved